modular_skyrat/modules/vore/code/obj/belly.dm

```diff
@@ -0,0 +1,304 @@
+/obj/vbelly
+	name = "belly"
+	desc = ""
+	invisibility = INVISIBILITY_MAXIMUM
+	var/datum/gas_mixture/air_contents = new()
```

Comment:
Ditch this entirely, mobs should be handling all of their air stuff on their own, which they do.

Reply:
mechanically, if we are in a point where the air around the uh, container mob, is unsafe to be in, the stored mobs also should be

Reply:
What if the container mob is space-proof and goes on a space walk? Or like, the container mob wears internals when there's plasma/CO2 in the air? I think isolating it isn't a bad idea, since people will probably be miffed about needing to wear internals/a space suit when they're 99.9% of the time just trying to be horny. Skyrat seems really capable of handling things administratively if it's abused anywho. (Oh, the stories of Citadel combat vore...)

Reply:
If there is a situation where the atmos of the location matters mechanically, then the scene has likely ended and frankly that level of edge case is not worth supporting